docs/projects/lottie/layers/rectangle_layer.rst

```diff
@@ -0,0 +1,46 @@
+Rectangle Layer
+===============
+
+Table of Mappings
+-------------------
+
++---------------------+---------------------------------------------------------------------------+
+|       Property      |                         Path to Property in lottie                        |
++=====================+===========================================================================+
+|       Z depth       |               Depends on ordering of layers in lottie format              |
++---------------------+---------------------------------------------------------------------------+
+|        Amount       | layers/fill.json -> “o” -> properties/[value.json OR valueKeyframed.json] |
++---------------------+---------------------------------------------------------------------------+
+|     Blend_method    |            layers/shape.json -> “bm” -> helpers/blendMode.json            |
++---------------------+---------------------------------------------------------------------------+
+|        Color        |                      layers/fill.json -> “ty” = “fl”                      |
```

Comment:
Can you explain “ty” = “fl”? (Perhaps in the note below)

Reply:
Done. :slightly_smiling_face: